app/models/burrito.rb

```diff
@@ -18,4 +18,14 @@ class Burrito < ApplicationRecord
   has_many(:ratings, { :class_name => "Rating", :foreign_key => "burrito_id", :dependent => :destroy })
   belongs_to(:restaurant, { :required => false, :class_name => "Restaurant", :foreign_key => "restaurant_id", :counter_cache => true })
   has_many(:raters, { :through => :ratings, :source => :rater })
+
+  def avg_rating
+    ratings = Rating.where({ :burrito_id => self.id })
```

Comment:
You can use the association you defined instead

```suggestion
    ratings = self.ratings
```